#! /bin/sh /usr/share/dpatch/dpatch-run
## 38_fix_libexec.dpatch by Jan Wagner <waja@cyconet.org>
##
## DP: fix library path

@DPATCH@
diff -urNad nagios-plugins-1.4.12~/contrib/check_linux_raid.pl nagios-plugins-1.4.12/contrib/check_linux_raid.pl
--- nagios-plugins-1.4.12~/contrib/check_linux_raid.pl	2007-08-28 05:19:45.000000000 +0200
+++ nagios-plugins-1.4.12/contrib/check_linux_raid.pl	2008-06-06 16:16:23.000000000 +0200
@@ -23,7 +23,7 @@
 #	  WARNING md0 status=[UUU_U], recovery=46.4%, finish=123.0min
 
 use strict;
-use lib "/usr/local/nagios/libexec";
+use lib "/usr/lib/nagios/plugins";
 use utils qw(%ERRORS);
 
 # die with an error if we're not on Linux
